install.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
 
+# Redirect output to a logfile
+exec > >(tee -a logs/mlbled.log) 2>&1
```

Comment:
IMO it makes sense to have a separate install.log from the runtime log, but I think it’s fine if they are combined

Reply:
No strong feelings either way, was hoping it makes it easier for users to send us relevant logs since they will all be in one spot

Reply:
Fair enough